src/System.Data.SqlClient/src/System/Data/SqlClient/SqlDelegatedTransaction.cs

```diff
@@ -397,7 +397,7 @@ public void SinglePhaseCommit(SinglePhaseEnlistment enlistment)
 
         // Event notification that transaction ended. This comes from the subscription to the Transaction's
         //  ended event via the internal connection. If it occurs without a prior Rollback or SinglePhaseCommit call,
-        //  it indicates the transaction was ended externally (generally that one the the DTC participants aborted
+        //  it indicates the transaction was ended externally (generally that one the DTC participants aborted
```

Comment:
The fix looks wrong... I believe it should be "generally that one of the DTC participants".